components/automate-ui/src/app/modules/team/team-details/team-details.component.html

```diff
@@ -78,9 +78,9 @@
         </form>
         <chef-button [disabled]="isLoadingTeam || !updateNameForm.valid || !updateNameForm.dirty" primary inline
           (click)="saveTeam()" data-cy="team-details-submit-button">
-          <chef-loading-spinner *ngIf="saving"></chef-loading-spinner>
-          <span *ngIf="saving">Saving...</span>
-          <span *ngIf="!saving">Save</span>
+          <chef-loading-spinner *ngIf="saveInProgress"></chef-loading-spinner>
+          <span *ngIf="saveInProgress">Saving...</span>
+          <span *ngIf="!saveInProgress">Save</span>
```

Comment:
PR bonus: this was broken--would never see "Save" -> "Saving..." -> "Save" on team details due to wrong variable.